resources/kcp/charts/keda/README.md

```diff
@@ -0,0 +1,147 @@
+| `prometheus.metricServer.podMonitor.interval`              | Scraping interval for metric server using podMonitor crd (prometheus operator) | ``
+| `prometheus.metricServer.podMonitor.scrapeTimeout`         | Scraping timeout for metric server using podMonitor crd (prometheus operator) | ``
+| `prometheus.metricServer.podMonitor.namespace`             | Scraping namespace for metric server using podMonitor crd (prometheus operator) | ``
+| `prometheus.metricServer.podMonitor.additionalLabels`      | Additional labels to add for metric server using podMonitor crd (prometheus operator) | `{}`
+| `prometheus.operator.enabled`                              | Enable keda operator prometheus metrics expose | `false`
+| `prometheus.operator.port`                                 | HTTP port used for exposing keda operator prometheus metrics | `9022`
+| `prometheus.operator.path`                                 | Path used for exposing keda operator prometheus metrics | `/metrics`
+| `prometheus.operator.podMonitor.enabled`                   | Enable monitoring for keda operator using podMonitor crd (prometheus operator) | `false`
+| `prometheus.operator.podMonitor.interval`                  | Scraping interval for keda operator using podMonitor crd (prometheus operator) | ``
+| `prometheus.operator.podMonitor.scrapeTimeout`             | Scraping timeout for keda operator using podMonitor crd (prometheus operator) | ``
+| `prometheus.operator.podMonitor.namespace`                 | Scraping namespace for keda operator using podMonitor crd (prometheus operator) | ``
+| `prometheus.operator.podMonitor.additionalLabels`          | Additional labels to add for keda operator using podMonitor crd (prometheus operator) | `{}`
+| `prometheus.operator.prometheusRules.enabled`              | Enable monitoring for keda operator using prometheusRules crd (prometheus operator) | `false`
+| `prometheus.operator.prometheusRules.namespace`            | Scraping namespace for keda operator using prometheusRules crd (prometheus operator) | ``
+| `prometheus.operator.prometheusRules.additionalLabels`     | Additional labels to add for keda operator using prometheusRules crd (prometheus operator) | `{}`
+| `prometheus.operator.prometheusRules.alerts`               | Additional alerts to add for keda operator using prometheusRules crd (prometheus operator) | `[]`
+| `volumes.keda.extraVolumes`                                | Extra volumes for keda deployment | `[]`
+| `volumes.keda.extraVolumeMounts`                           | Extra volume mounts for keda deployment | `[]`
+| `volumes.metricsApiServer.extraVolumes`                    | Extra volumes for metric server deployment | `[]`
+| `volumes.metricsApiServer.extraVolumeMounts`               | Extra volume mounts for metric server deployment | `[]`
+
+Specify each parameter using the `--set key=value[,key=value]` argument to

```

Comment:
```suggestion
Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`.

```

Reply:
See first comment. This was resolved after our Slack discussion.